### PR TITLE
Working Code samples for "Flutter for React native developers" doc.

### PR DESCRIPTION
**Pull request by Samaritan1011001**

### CODE REVIEW

**Comment by Hixie** [comment]
Please do not commit the grade wrapper jar or graddle wrapper script.

File: animations/flutterfade/test/widget_test.dart [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    expect(find.text('1'), findsNothing);
+
+    // Tap the '+' icon and trigger a frame.
+    await tester.tap(find.byIcon(Icons.add));
+    await tester.pump();
+
+    // Verify that our counter has incremented.
+    expect(find.text('0'), findsNothing);
+    expect(find.text('1'), findsOneWidget);
+  });
```
Comment (Hixie) [inline animations/flutterfade/test/widget_test.dart]: it seems unlikely that this test will pass.

**Comment by Hixie** [comment]
This PR is too big to be reasonable reviewable on GitHub. Can we split it into several so that each individual part can be reviewed separately?

File: animations/flutterfade/test/widget_test.dart (hunk 1 above)
Comment (Samaritan1011001) [reply animations/flutterfade/test/widget_test.dart]: We have not written tests for any of the samples. Should I write them for all the samples? What about the React Native counter parts?

**Comment by Samaritan1011001** [comment]
I agree. How do you want me to split them? Do you want to review each main folder individually or in groups of five since there are 15 of them.

File: animations/flutterfade/test/widget_test.dart (hunk 1 above)
Comment (Hixie) [reply animations/flutterfade/test/widget_test.dart]: I don't have an opinion on that either way, but we shouldn't have tests that fail. :-)

**Comment by Hixie** [comment]
I don't have a good understanding of what this patch is or how it's organised so it's hard for me to suggest a way to submit it that's more reviewable. Ideally each PR should be self-contained so that it makes sense on its own, building on what came before.

**Comment by Samaritan1011001** [comment]
I have divided the samples into 3 and created a PR with the description to each sample in PR description. Let me know if three of these per PR is good enough after which I'll create the rest of the PRs. The link to the PR is [here](https://github.com/flutter/flutter_for_react_native/pull/2)

**Comment by Samaritan1011001** [comment]
To make sure we are on the same page here. This repo was created so we can add the working code samples that we wrote for the doc. Code snippets from these samples are used in the "Flutter for React Native" section that is to be published on the flutter.io website. @mjohnsullivan and Wm have reviewed the document.

**Comment by Samaritan1011001** [comment]
I have created another PR [here](https://github.com/flutter/flutter_for_react_native/pull/3) from the  GeekyAnts account and closed this one. We can discuss the rest there.

**Pull request closed without merging**